voicebox/__main__.py

```diff
@@ -6,14 +6,21 @@
 
 
 logging.basicConfig(
-    level=logging.DEBUG,
+    level=logging.INFO,
     format='%(asctime)s %(name)s %(levelname)-8s  %(message)s',
     datefmt='(%H:%M:%S)'
 )
 
 
 def main():
 
+    """
+    Main function to start system
+
+    provides a command line interface for interracting
+    with the voicebox APIs.
+    """
+
     username = input("Username: ")
 
     ip = extract_ip()
```

Comment:
The `main` function could benefit from error handling. For instance, if the `extract_ip` function fails to retrieve the IP address, the program will crash. Implementing error handling can improve the robustness of the code. [important]